src/sphinx_book_theme/assets/styles/abstracts/_variables.scss

```diff
@@ -50,9 +52,15 @@ $border-thin: 1px solid rgba(0, 0, 0, 0.1);
   --pst-color-admonition-important: var(--pst-color-info);
 
   --pst-color-warning: 245, 162, 82;
+  --pst-color-admonition-caution: var(--pst-color-warning);
+  --pst-color-admonition-attention: var(--pst-color-warning);
+
 
   --pst-color-danger: 230, 101, 129;
```

Comment:
Didn't the `pst` variables get re-worked to be full `rgb(` values now instead of just numbers?

Reply:
That is true. But, I will have to rewrite this whole file for that then for consistency. As other variables are using numbers. You reckon I should do it in this PR?

Reply:
I think so, otherwise we are going to have two different CSS variable structures, one that uses `rgb(` and another that just defines numbers. I think we should just standardize on using `rgb(`